src/marten/http/session/store/base.cr

```diff
@@ -9,12 +9,27 @@ module Marten
           @accessed : Bool = false
           @modified : Bool = false
           @session_hash : SessionHash? = nil
+          @expires : Int32? = nil
 
           getter session_key
 
           def initialize(@session_key : String?)
           end
 
+          def expires=(value : Int32)
```

Comment:
Can we name this method (and the associated instance variable) `#expires_at`?

Reply:
Sure